Doppler.Sap/Mappers/Billing/BillingForArMapper.cs

```diff
@@ -99,5 +99,11 @@ public SapSaleOrderModel MapDopplerBillingRequestToSapSaleOrder(BillingRequest b
 
             return sapSaleOrder;
         }
+
+        public SapIncomingPaymentModel MapSapIncomingPayment(int docEntry, string cardCode, decimal docTotal)
+        {
+            //Is not implemented because at the moment is not necessary the send the Payment to SAP
+            throw new System.NotImplementedException();
```

Comment:
what billing could be to pass with this line in the future, you have an example?

Reply:
I don't understand the question. Currently it mapper hasn't implementation becuase for the AR SAP we don't send to SAP the payment.

Reply:
Ok I understand